### ISSUE TITLE
deleteSession! error on sessionConnection_ALL

### ISSUE THREAD

**Issue by dehann** [NavAbility/NavAbilitySDK.jl#252]
```julia
julia> deleteSession!(DFGClient(userLabel, robotLabel, sessionLabel; apiUrl))
ERROR: GraphQLError: Request to server failed

Message:

Field "sessionConnection_ALL" is not defined by type "BlobEntryWhere".

Message:

Field "sessionConnection_ALL" is not defined by type "BlobEntryWhere".
Stacktrace:
 [1] checkbodyforerrors
   @ ~/.julia/packages/GraphQLClient/Wm5fb/src/http_execution.jl:129 [inlined]
 [2] _execute(endpoint::String, execution_string::String, headers::Dict{…}, output_type::Type{…}; retries::Int64, readtimeout::Int64, operation_name::Nothing, throw_on_execution_error::Bool)
   @ GraphQLClient ~/.julia/packages/GraphQLClient/Wm5fb/src/http_execution.jl:116
 [3] execute(endpoint::String, payload::Dict{…}, headers::Dict{…}, output_type::Type{…}; kwargs::@Kwargs{…})
   @ GraphQLClient ~/.julia/packages/GraphQLClient/Wm5fb/src/http_execution.jl:80
 [4] execute(client::GraphQLClient.Client, payload::Dict{…}, output_type::Type{…}; kwargs::@Kwargs{…})
   @ GraphQLClient ~/.julia/packages/GraphQLClient/Wm5fb/src/http_execution.jl:74
 [5] execute(client::GraphQLClient.Client, query::String, output_type::Type{…}; variables::Dict{…}, operation_name::Nothing, kwargs::@Kwargs{…})
   @ GraphQLClient ~/.julia/packages/GraphQLClient/Wm5fb/src/http_execution.jl:70
 [6] execute
   @ ~/.julia/packages/GraphQLClient/Wm5fb/src/http_execution.jl:69 [inlined]
 [7] deleteSession!(fgclient::DFGClient)
   @ NavAbilitySDK ~/.julia/dev/NavAbilitySDK/src/services/UserRobotSession.jl:179
 [8] top-level scope
   @ REPL[15]:1
Some type information was truncated. Use `show(err)` to see complete types.
```

**Comment by dehann** [comment]
```julia
julia> listSessionBlobEntries(DFGClient(userLabel, robotLabel, sessionLabel; apiUrl))
Symbol[]
```

**Comment by dehann** [comment]
this was already fixed in v0.7.0, I just had an older version of code.
